test/e2e/density.go

```diff
@@ -632,6 +661,48 @@ var _ = framework.KubeDescribe("Density", func() {
 			}
 		})
 	}
+
+	// Calculate total number of pods from each node's max-pod
+	It("[Feature:ManualPerformance] should allow running maximum capacity pods on nodes", func() {
+		totalPods = 0
+		for _, n := range nodes.Items {
+			totalPods += int(n.Status.Capacity.Pods().Value())
+		}
+		totalPods -= framework.WaitForStableCluster(c, masters)
+
+		fileHndl, err := os.Create(fmt.Sprintf(framework.TestContext.OutputDir+"/%s/pod_states.csv", uuid))
```

Comment:
You probably want to have a flag or env var that enables/disables writing output files (when we had this kind of behavior previously it ended in a number of disk full problems).